<Hotfix: Villager AI Movement>

projojoboy opened the pull request:
Fix Movement bug
Fix House nav mesh obstacle size

projojoboy committed:
<Hotfix: Villager AI Movement>
Fix Movement bug
Fix House nav mesh obstacle size

empireboy commented on Assets/Scripts/Features/Gameplay/AI/Actions/WanderingAction.cs:
```diff
@@ -7,9 +7,6 @@
-    // Reference to the NavMeshAgent component.
-    private NavMeshAgent _navMeshAgent;
-
     // The destination the villager has to wander to.
     private Vector3 _targetDestination;
 
```
Every villager will use the same _targetDestination here

projojoboy committed:
<Hotfix: Villager AI Movement>
Add Target destination in State Controller

waffel183 committed:
<Hotfix: Villager AI Movement>
Fix Nav Mesh Obstacle size on Buildings

waffel183 committed:
<Hotfix: Villager AI Movement>
Fix Villagers floating above ground

waffel183 committed:
<Hotfix: Villager AI Movement>
Remove Meta files for removed folders

waffel183 committed:
<Hotfix: Villager AI Movement>
Fix State dicision chance
Fix Idle animation
Fix Villager Prefab
Fix NavMeshAreas to reachable locations

waffel183 committed:
<Hotfix: Villager AI Movement>
Fix Merge Conflict
